K8s: DOC-1683 incorrect command to remove replace password (#2173)
RedisLabs/redislabs-docs: content/kubernetes/security/manage-rec-credentials.md

aliases: 
  /platforms/kubernetes/concepts/manage_REC_credentials/
  /platforms/kubernetes/concepts/manage_REC_credentials.md
  /platforms/kubernetes/security/manage_REC_credentials/
  /platforms/kubernetes/security/manage_REC_credentials.md
  /kubernetes/security/manage_REC_credentials.md
  /kubernetes/security/manage_REC_credentials/
 
---
The Redis Enterprise Software on Kubernetes uses a custom resource called [`RedisEnterpriseCluster`]({{<relref "/kubernetes/reference/cluster-options.md">}}) to create a Redis Enterprise cluster (REC). During creation it generates random credentials for the operator to use. The credentials are saved in a Kubernetes (K8s) [secret](https://kubernetes.io/docs/concepts/configuration/secret/). The secret name defaults to the name of the cluster.

{{<note>}}
This procedure is only supported for operator versions 6.0.20-12 and above.
{{</note>}}

## Retrieve the current username and password

The credentials can be used to access the Redis Enterprise admin console or the API. Connectivity must be configured to the REC [pods](https://kubernetes.io/docs/concepts/workloads/pods/) using an appropriate service (or port forwarding).

> 1. Inspect the random username and password created by the operator during creation, use the `kubectl get secret` command.

Reviewer: ```suggestion
1. Inspect the random username and password created by the operator during creation with the `kubectl get secret` command.
```

    ```sh
    kubectl get secret rec -o jsonpath='{.data}'
    ```

>     The output will be the encoded password and username similar to the example below.

Reviewer: ```suggestion
    The command outputs the encoded password and username, similar to the example below.
```
